helpers/update-site/documentation.js

```diff
@@ -725,7 +725,8 @@ const createHintCategories = (files) => {
 };
 
 const updateChangelog = async () => {
-    const files = await globby([`${constants.dirs.NODE_MODULES}/{hint,vscode-webhint,@hint/{${resources.join(',')},configuration-all/node_modules/{vscode-webhint,@hint/{${resources.join(',')}}}-*}}/CHANGELOG.md`]);
+    /** All packages should be hoisted so no need to look inside `configuration-all` */
+    const files = await globby([`${constants.dirs.NODE_MODULES}/{hint,vscode-webhint,@hint/{${resources.join(',')}}-*}/CHANGELOG.md`]);
```

Comment:
I think there are a couple of more places that use a similar globby pattern that should be updated.

Reply:
I've updated the regex everywhere.